lib/actions/down.js

```diff
@@ -9,12 +9,18 @@ const hasCallback = require('../utils/has-callback');
 
 module.exports = async (db, client) => {
   const downgraded = [];
-  const statusItems = await status(db);
+  const { saveFileContents, changelogCollectionName } = await config.read();
+  const changelogCollection = db.collection(changelogCollectionName);
+  const statusItems = await status(db, saveFileContents);
   const appliedItems = statusItems.filter(item => item.appliedAt !== "PENDING");
   const lastAppliedItem = _.last(appliedItems);
 
   if (lastAppliedItem) {
     try {
+      if (saveFileContents) {
```

Comment:
If we are using saved file contents, when doing a down migration we use the migration that was saved in the database, not the one read from the file system (which may be non existent). We do this by overwriting the file in the filesystem with the content read from the database